src/wooting-usb.c

```diff
@@ -58,11 +60,17 @@ void wooting_usb_set_disconnected_cb(void_cb cb) {
 }
 
 bool wooting_usb_is_wooting_one() {
-	return is_wooting_one;
+	if (!wooting_usb_meta.connected) wooting_usb_find_keyboard();
```

Comment:
true. will change it tomorrow. it certainly is reasonable. also since i figured out how to differ between ansi and iso too

Reply:
Maybe turn that into some function like `wooting_usb_get_info` that returns an object structure with information about the keyboard.

The JSON representation of what it would return could look like this for a Wooting 2: ```json
{
	generation: 2,
	iso: false,
	serial: '000000000000000000', // if we can access that
	firmware_version; '1.24.3'
}
```